test/libsolidity/smtCheckerTests/complex/slither/const_state_variables.sol

```diff
@@ -53,6 +53,5 @@ contract MyConc{
 // ----
 // Warning: (773-792): This declaration shadows an existing declaration.
 // Warning: (1009-1086): Function state mutability can be restricted to view
-// Warning: (874-879): Underflow (resulting value less than 0) happens here.
-// Warning: (874-879): Overflow (resulting value larger than 2**256 - 1) happens here.
+// Warning: (985-1002): Underflow (resulting value less than 0) happens here.
```

Comment:
Well, there are implicit constructors and state variable initialization, which are also part of this PR.
Yes, in `contract MyConc`, since the state variables assignments were not taken into account before, `uint should_be_constant_2 = A + 1;` used to overflow. Now that it knows that `A = 1`, it doesn't anymore.